docs/src/main/paradox/index.md

```diff
@@ -1,70 +1,85 @@
-Akka Quickstart with Scala
-==========================
+# Akka Quickstart with Scala
+ 
+Akka is a toolkit and runtime for building highly concurrent, distributed, and fault-tolerant event-driven applications on the JVM. Akka can be used with both Java and Scala.
+This guide introduces Akka by describing the Scala version of the Hello World example. If you prefer to use Akka with Java, switch to the [Akka Quickstart with Java guide](http://developer.lightbend.com/start/?group=akka&project=akka-java-seed). 
 
-[Akka](http://akka.io) is a toolkit and runtime for building highly concurrent,
-distributed, and fault-tolerant event-driven applications on the JVM. Akka can be used with both Java
-and Scala.
+Actors are the unit of execution in Akka. The Actor model as an abstraction that makes it easier to write correct concurrent, parallel and distributed systems. The Hello World example illustrates Akka basics. Within 30 minutes, you should be able to download and run the example and use this guide to understand how the example is constructed. This will get your feet wet, and hopefully inspire you to dive deeper into the wonderful sea of Akka!
 
-This tutorial will guide you through the some basic but fundamental Akka building blocks.
+## Downloading the example 
```

Comment:
She recommends using this style.  "you might want to use sentence case everywhere, this seems to be the more modern (and international) trend)"